src/Metadata/Resource/ResourceMetadataCollection.php

```diff
@@ -89,9 +89,9 @@ public function getOperation(?string $operationName = null, bool $forceCollectio
         }
 
         // Idea:
-//         if ($metadata) {
-//             return (new class extends HttpOperation {})->withResource($metadata);
-//         }
+        //         if ($metadata) {
+        //             return (new class extends HttpOperation {})->withResource($metadata);
+        //         }
```

Comment:
Hey @PierreRebeilleau, remember me ? ;)

I think this has nothing to do with your PR. So maybe I'm wrong but you might have to revert it.
I actually thought of doing a dedicated PR for this because it affect all the created branch

Reply:
I already did that this needs rebase

Reply:
Hey @MarvinCourcier , yes i remember you ! Glad to see you around, ty i'm gonna rebase :)